### ISSUE TITLE
How to build a training set?

### ISSUE THREAD

**Issue by ruru5697** [Bjoux2/DeepDTIs#1]
I read your paper "Deep learning-based drug-target interaction prediction". It make me interested to use deep belief nets to predict drug-target interaction. And I use tensorflow to implement deep belief nets, following your idea and code.
But the nets' performance is worse. The strategy I build training set is: 
1. use approved drug-target interaction as positive sample, almost have 4500 samples.
2. random select drug and target who not in approved data set as negative sample, number of negative sample is same as positive.

And I use same strategy to build test set, using experimental data.
After train the nets, I use test set to evaluate nets, and I see almost drug-target records are predict as positive, only a few records predict as negative (maybe 200 of 3000).
I read the code in file 'code/load_data.py', and the function 'load_labeled_data' is need a parameter for a pkl format file. But I can't find this file or convert data code in Github.
So how to build a training set for this nets?
Thanks.

**Comment by hebaebrahim** [comment]
> I read your paper "Deep learning-based drug-target interaction prediction". It make me interested to use deep belief nets to predict drug-target interaction. And I use tensorflow to implement deep belief nets, following your idea and code.
> But the nets' performance is worse. The strategy I build training set is:
> 
> 1. use approved drug-target interaction as positive sample, almost have 4500 samples.
> 2. random select drug and target who not in approved data set as negative sample, number of negative sample is same as positive.
> 
> And I use same strategy to build test set, using experimental data.
> After train the nets, I use test set to evaluate nets, and I see almost drug-target records are predict as positive, only a few records predict as negative (maybe 200 of 3000).
> I read the code in file 'code/load_data.py', and the function 'load_labeled_data' is need a parameter for a pkl format file. But I can't find this file or convert data code in Github.
> So how to build a training set for this nets?
> Thanks.

please if you have the training data set for this paper  
upload here for uses in my research 
thanks

**Comment by 89-oniichan** [comment]
Man if you have dataset and Code please send me or give me a cloud link please

**Comment by ruru5697** [comment]
> Man if you have dataset and Code please send me or give me a cloud link please

I crawl the dataset from drugbank about 5 year ago, it maybe out of date for reseach.
